src/plumpy/process_comms.py

```diff
@@ -401,28 +401,24 @@ def play_all(self) -> None:
         """
         self._communicator.broadcast_send(None, subject=Intent.PLAY)
 
-    def kill_process(self, pid: 'PID_TYPE', msg: Optional[MessageType] = None) -> kiwipy.Future:
+    def kill_process(self, pid: 'PID_TYPE', msg_text: Optional[str] = None, force_kill: bool = False) -> kiwipy.Future:
```

Comment:
I'd say, let's add this flag only in the other PR https://github.com/aiidateam/plumpy/pull/288
Which is specifically for `force_kill` feature, and offers more changes.

I though the scope of the current PR, was to implement message interface, only.

Reply:
This I agree, I'll remove it and leave it for your PR.